.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    environment: ci
+    strategy:
+      matrix:
+        ruby-version: ['2.7', '3.0.6', '3.1', '3.2']
```

Comment:
Not for this PR, but once this is merged, we should consider going back to `3.0` from `3.0.6`. This was changed in #29 due to rvm on Travis, but GHA doesn't use rvm, so that probably likely doesn't exist. cc @agrare